File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/sak/Sak.java
package no.nav.melosys.eessi.integration.sak;

import java.time.ZonedDateTime;

import lombok.*;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Sak {

  private String id;
  private String tema;
  private String applikasjon;
  private String aktoerId;
  private String orgnr;
  private String fagsakNr;
  private String opprettetAv;
  private ZonedDateTime opprettetTidspunkt;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/sak/Sak.kt
package no.nav.melosys.eessi.integration.sak

import java.time.ZonedDateTime

data class Sak(
    var id: String? = null,
    var tema: String? = null,
    var applikasjon: String? = null,
    var aktoerId: String? = null,
    var orgnr: String? = null,
    var fagsakNr: String? = null,
    var opprettetAv: String? = null,
    var opprettetTidspunkt: ZonedDateTime? = null
)
